.gitignore

```diff
@@ -9,3 +9,4 @@ target
 *.iws
 .idea
 .DS_Store
+.vscode/
```

Comment:
Hi, @dirtysalt . This is orthogonal change from the proposed document fix. Apache ORC recommends that each commit should have one theme. Could you revert this change from this PR?

Reply:
Yes. Sorry, I don't expect this PR  will trace my master branch commit automatically. Reverted already.